images/mod-tile/README.md

```diff
@@ -12,24 +12,23 @@ renderd uses a software library who does the actual rendering called **Mapnik**
 After rendering a tile, it gets cached on disk for fast serving using mod_tile. As OSM data is constantly updated, improved and changed, a mechanism is needed to correctly expire the cache to ensure updated tiles get re-rendered.
 Tiles will cached on the `/var/lib/mod_tile` directory by default.
 Apache serves the files as if they were present under `/[MOD_TILE_HOST]:[MOD_TILE_PORT]/[MOD_TILE_PATH]/Z/X/Y.png`
-mod_tile is able to hold multiple mapnik style sheets. At the moment only the **openstreetmap-carto** stylesheet configuration is being used. with every import or update of tiles using the `osm2pgsql` tool style and database tag scheme needs to be specified.
+mod_tile is able to hold multiple mapnik style sheets. At the moment only the **openstreetmap-carto** stylesheet configuration is being used. With every import or update of tiles using the `osm2pgsql` tool style and database tag scheme needs to be specified.
 
 For tiles to reflect the last update they need to be deleted or tagged as expired to force their being rerendered.
 Expired tiles are being re-rendered in a loop every configured interval (`RENDER_EXPIRED_TILES_INTERVAL`).
-Every replication (osmChange file) created by the `replication-job` container will be processed by `osm2pgsql` which in turn will send the expired tiles list to `/mnt/expired/` directory. The expired tiles will be rendered by mod_tile.
-The sequence number of last rendered replication will be stated in `/mnt/expired/state.txt` as `lastRendered` key.
+Every replication (osmChange file) created by the `replication-job` container will be processed by `osm2pgsql` which in turn will send the expired tiles list to `EXPIRED_DIRECTORY` directory. The expired tiles will be rendered by mod_tile.
+The sequence number of last rendered replication will be stated in `EXPIRED_DIRECTORY/renderedState.txt` as `lastRendered` key.
```

Comment:
please change to `[EXPIRED_DIRECTORY]/renderedState.txt` so the README.md will be uniformal

Reply:
done